### DISCUSSION TITLE
Cannot fuel charcoal forge with mineral coal

### DISCUSSION THREAD

**Discussion by Soadreqm** [cataclysmbn/Cataclysm-BN#2056] [General]
# Describe the bug

The furniture-type charcoal forges (called "forge" and "rock forge") do not accept non-wood coal as fuel.

# Steps To Reproduce

1. construct a forge, either from rocks or by placing a charcoal forge item
2. spawn coal (not charcoal)
3. 'e'xamine forge
4. cannot reload

# Expected behavior

Furniture forges should accept coal in place of charcoal, like the "charcoal forge" item does.

# Versions and configuration

- OS: Windows 10 Pro N 2009
- Game Version: b153e66 [64-bit]
- Graphics Version: Tiles
- Game Language:  []
- Mods loaded: [
    Bright Nights [dda],
    Disable NPC Needs [no_npc_food],
    Simplified Nutrition [novitamins],
    Elevated bridges [elevated_bridges],
    No Rail Stations [No_Rail_Stations],
    Fuji's Military Profession Pack [fuji_mpp],
    Mythical Martial Arts [MMA]
]

**Comment by chaosvolt** [comment]
From what I recall, the issue seems to be that reloadable furniture can only ever accept the default ammo for their ammotype, not all available ammo options. So for example, the coal ammotype defines charcoal as the default ammo, so only charcoal can be used.
